pkgs/development/python-modules/pyzstd/default.nix

```diff
@@ -5,17 +5,19 @@
   pytestCheckHook,
   setuptools,
   zstd-c,
+  typing-extensions,
```

Comment:
You should keep the existing alphabetical ordering.

Reply:
Not all packages do that, but thanks.

Reply:
That's why I said *keep*.